deploy/static/starboard.yaml

```diff
@@ -822,7 +822,7 @@ metadata:
     app.kubernetes.io/version: "0.15.2"
     app.kubernetes.io/managed-by: kubectl
 data:
-  trivy.imageRef: "docker.io/aquasec/trivy:0.24.2"
+  trivy.imageRef: "docker.io/aquasec/trivy:0.25.2"
```

Comment:
I think we should also add trivy.dbRepository property here, otherwise you'll get errors: `property trivy.dbRepository not set`